source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Infrastructure.Core/Cim/ValidationErrors/CimValidationErrorTextTemplateMessages.cs

```diff
@@ -157,6 +157,14 @@ public static class CimValidationErrorTextTemplateMessages
         public const string NumberOfPointsMatchTimeIntervalAndResolutionText =
             "The number of prices received does not match the expected number of prices given the time interval and resolution provided.";
 
+        [ErrorMessageFor(ValidationRuleIdentifier.ChargeNameIsRequired)]
+        public const string ChargeNameRequiredErrorText =
+            "Charge name is missing: transaction can not be processed for document {{DocumentId}}";
+
+        [ErrorMessageFor(ValidationRuleIdentifier.ChargeDescriptionIsRequired)]
+        public const string ChargeDescriptionRequiredErrorText =
+            "Charge description is missing: transaction can not be processed for document {{DocumentId}}";
+
```

Comment:
I suggest to keep it simple as the rejection also contains a reference to the original operation.

```suggestion
        [ErrorMessageFor(ValidationRuleIdentifier.ChargeNameIsRequired)]
        public const string ChargeNameRequiredErrorText =
            "Charge name is missing.";

        [ErrorMessageFor(ValidationRuleIdentifier.ChargeDescriptionIsRequired)]
        public const string ChargeDescriptionRequiredErrorText =
            "Charge description is missing.";

```